rs/backend/src/tvl.rs

```diff
@@ -1,18 +1,30 @@
 use crate::{
     canisters::{exchange_rate_canister, governance},
-    constants::NANOS_PER_UNIT,
+    constants::{E8S_PER_UNIT, NANOS_PER_UNIT},
     spawn,
     state::STATE,
     time,
     timer::{set_timer, set_timer_interval},
 };
+use candid::{CandidType, Nat};
 use std::time::Duration;
 
 pub mod state;
 
 const XRC_MARGIN_SECONDS: u64 = 60 * 5;
 const UPDATE_INTERVAL_SECONDS: u64 = 6 * 60 * 60; // 4 times a day
 
+#[derive(CandidType, Debug, PartialEq)]
+pub struct TvlResult {
+    pub tvl: Nat, // Total Value Locked in whole USD.
+    pub time_sec: Nat,
```

Comment:
NCR; just for your consideration

```suggestion
    pub exchange_rate_timestamp_seconds: Nat,
```

Reply:
(Intuitively, there's semantic difference between the words time and timestamp)

Reply:
same